dbms/src/Functions/FunctionsHashing.h

```diff
@@ -842,6 +896,10 @@ class FunctionAnyHash : public IFunction
         else if (which.isString()) executeString<first>(icolumn, vec_to);
         else if (which.isFixedString()) executeString<first>(icolumn, vec_to);
         else if (which.isArray()) executeArray<first>(from_type, icolumn, vec_to);
+        else if (which.isDecimal32()) executeDecimal<Decimal32, first>(icolumn, vec_to);
+        else if (which.isDecimal64()) executeDecimal<Decimal64, first>(icolumn, vec_to);
+        else if (which.isDecimal128()) executeDecimal<Decimal128, first>(icolumn, vec_to);
+        else if (which.isUUID()) executeUUID<first>(icolumn, vec_to);
```

Comment:
Could you please also add `executeGeneric` that will call virtual method `getDataAt` of `IColumn`?